apps/web-mzima-client/src/app/shared/components/fragments/share-and-donate/share-and-donate.component.scss

```diff
@@ -17,19 +17,29 @@
 .menu {
   &__footer-container {
     position: fixed;
-    bottom: 0;
-    left: 0;
-    width: 100%;
+    // bottom: 0;
+    // left: 0;
+    // width: 100%;
+    min-width: 180px;
```

Comment:
Always delete code entirely rather than commenting it out. If we want ot look at the history, we can always check github :)

Reply:
@Angamanga 
I've deleted the commented out line of code